doc/install_astra_camera.rst

```diff
@@ -78,7 +78,7 @@ Then, edit ``/usr/lib/pkgconfig/libopenni2.pc`` to be like below::
 
 .. code-block:: bash
 
-  cd ~/CATKIN_WS/src
+  cd ~/YOUR_CATKIN_WS/src
```

Comment:
How about `cd <your catkin workspace>/src`?

Reply:
I think it's good.
I rebased our previous commits.